aerofiles/seeyou/reader.py

```diff
@@ -78,7 +78,7 @@ def decode_waypoint(self, fields):
         if num_fields < 11:
             raise ParserError('Not enough fields provided. Expecting at minimum following 11 fileds:\nname,code,country,lat,lon,elev,style,rwdir,rwlen,freq,desc')
 
-        if num_fields > 13:
+        if num_fields > 15:
```

Comment:
how are you getting to 15? it looks like `userdata` and `pics` were already part of the previous 13, and by adding `rwwidth` we're now at 14, right?